spec/features/charge_and_pass_libraries_spec.rb

```diff
@@ -21,7 +21,7 @@
   background do
     # We look up the user
     stub_swipecard_search(user_swipecard, user)
```

Comment:
This spec also has a 'default_tube_printer' defined. Can that be removed since its not used?